src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -311,7 +374,7 @@ def load_arguments(self, _):
     # Services
     with self.argument_context('cosmosdb service') as c:
         c.argument('account_name', completer=None, options_list=['--account-name', '-a'], help='Name of the Cosmos DB database account.', id_part=None)
-        c.argument('resource_group_name', completer=None, options_list=['--resource-group-name', '-g'], help='Name of the resource group of the database account.', id_part=None)
+        c.argument('resource_group_name', completer=None, options_list=['--resource-group', '-g'], help='Name of the resource group of the database account.', id_part=None)

```

Comment:
May I know why deleting `--resource-group-name`? You can add extra `--resource-group` and keep `--resource-group-name` so that there won't be any breaking change

Reply:
@evelyn-ys Reverted the commit. This PR does not touch on this command group anyway.